src/modules/inspect/process-list/process-list.ts

```diff
@@ -114,10 +111,15 @@ export class ProcessList {
   public async updateCorrelationList(): Promise<void> {
     try {
       const correlations: Array<DataModels.Correlations.Correlation> = await this.getAllActiveCorrelations();
-      const correlationListWasUpdated: boolean = JSON.stringify(correlations) !== JSON.stringify(this._correlations);
-
+      const correlationListWasUpdated: boolean = JSON.stringify(correlations.sort(this.sortCorrelations)) !== JSON.stringify(this._correlations);
```

Comment:
Readd the Newline to increase readability & Add missing underscore

```suggestion
      const correlationListWasUpdated: boolean = JSON.stringify(correlations.sort(this._sortCorrelations)) !== JSON.stringify(this._correlations);

```